ws-nextjs-app/pages/[service]/new_live/[id]/live.test.tsx

```diff
@@ -35,6 +37,18 @@ describe('Live Page', () => {
     ).toBeInTheDocument();
   });
 
+  it('should render the posts', async () => {
+    await act(async () => {
+      render(<Live pageData={mockPageData} />);
+    });
+
+    expect(screen.getAllByText('Breaking news')[0]).toBeInTheDocument();
```

Comment:
Do you need to do the await act?

Would this work instead?

Also - maybe worth a second opinion on this - my preference is to return the assertion function from the render, instead of using `screen` - although I have seen both techniques used...

```suggestion
    const { getByText } = render(<Live pageData={mockPageData} />);

    expect(screen.getAllByText('Breaking news')[0]).toBeInTheDocument();
```

Reply:
This is another good question which I put in the Si tech channel. The reason we're using `await act` is covered in Aaron's PR - https://github.com/bbc/simorgh/pull/10818#discussion_r1186100651

Reply:
Ah right - if the tests need it, then that's absolutely fine. I didn't realise that there was different behaviour between NextJS & Express, so thanks for the heads up!